res/isabelle_settings

```diff
@@ -9,36 +9,18 @@ init_components "$USER_HOME/.isabelle/contrib" "$ISABELLE_HOME/Admin/components/
 init_components "$USER_HOME/.isabelle/contrib" "$ISABELLE_HOME/Admin/components/bundled"
 
 # 64 bit setup for large C proofs:
-ML_OPTIONS="-H 4096 --maxheap 16384"
+ML_OPTIONS="-H 4096 --maxheap 32768"
```

Comment:
It doesn't really matter since it just adds some noise to the history, but you throw away this change in the second commit.

Reply:
Agreed, nicer to clean it up.